includes/Logger.php

```diff
@@ -155,23 +169,14 @@ private function set_log_path() {
 		$wp_upload_dir       = wp_upload_dir( null, false );
 		$this->log_file_path = $wp_upload_dir['basedir'] . DIRECTORY_SEPARATOR;
 
-		if ( ! is_dir( $this->log_file_path ) ) {
-			wp_mkdir_p( $this->log_file_path );
-		}
-
 		$this->log_file_path_url = $wp_upload_dir['baseurl'] . DIRECTORY_SEPARATOR;
 	}
 
 	/**
 	 * Clear the log file.
 	 */
 	private function clear_log() {
```

Comment:
If it's not being used then we can remove it.

Reply:
That thing was used to be sure that the folder existed when trying to create the log file. But now is not longer necessary since we now try to delete it if it exists.